frontend/src/components/common/FilePicker.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { Form, Radio } from "antd";
+import { Field } from "redux-form";
+
+import FileUpload from "@/components/common/FileUpload";
+import { renderConfig } from "@/components/common/config";
+
+const propTypes = {
+  maxFileSize: PropTypes.string,
+  acceptedFileTypesMap: PropTypes.objectOf(PropTypes.string),
+  uploadUrl: PropTypes.string.isRequired,
+  existingFilesDropdown: PropTypes.arrayOf(
+    PropTypes.shape({
+      value: PropTypes.string,
+      label: PropTypes.string,
+    })
+  ).isRequired,
+  onSelectExisting: PropTypes.func.isRequired,
+};
+
+const defaultProps = {
+  maxFileSize: undefined,
+  acceptedFileTypesMap: undefined,
```

Comment:
I'm fine with this, but the pattern I've seen elsewhere in the codebase has been to use `null` instead

Reply:
I want to defer to the default props in FileUpload. Setting these to null won't use the defaults.